src/views/Jobs/Jobs.js

```diff
@@ -97,14 +97,14 @@ function Jobs({
                         appIcon={<AppIcon app="harvester-admin" />}
                         defaultWidth="fill"
                         padContent={false}
+                        height="92%"
```

Comment:
Why 92%?

Reply:
@MikeTaylor otherwise the pagination at the bottom of the screen would not be visible: 
![adminer](https://github.com/indexdata/ui-harvester-admin/assets/63545/6f30a1f9-49fd-4d53-a39e-682eb346a832)

I think this is due to the extra space at the top of the screen with the tabs: "Harvestables", "Jobs" and "Failed records" and the way panes fill out the page.

Reply:
That makes sense, but it's icky. There has to be better solution than guessing a slightly-smaller-than-100% number. If MCL doesn't interpret 100% as "100% of the vertical space actually availabe" something's wrong.

(To be clear: I am not at all suggesting this should have delayed the merging of the present PR; only flagging an underlying issue.)

Reply:
I agree with you. I think the panes are not calculated correctly when custom divs similar to: https://github.com/indexdata/ui-harvester-admin/blob/main/src/routes/SwitchRoute.js#L28-L48 are introduced.

This is a limitation I would say. Perhaps there is a way to restructure it a bit in ui-harvester-admin (wrap everything in Panes?) or this is a clear bug in stripes-components.

I haven't spent any time digging deeper though because that seems like a separate issue.

Reply:
Yeah, we would have to explore it a bit. I haven't seen a similar setup in other folio modules. Perhaps what could happen here is something similar to how the ui-inventory is setup (the tabs are above the search pane https://folio-snapshot.dev.folio.org/inventory)

Reply:
I can't see that working here: there are many tabs, and their names are long.